util/chplenv/printchplenv.py

```diff
@@ -77,7 +77,7 @@
     ChapelEnv('CHPL_COMM', RUNTIME | LAUNCHER | DEFAULT, 'comm'),
     ChapelEnv('  CHPL_COMM_SUBSTRATE', RUNTIME | LAUNCHER | DEFAULT),
     ChapelEnv('  CHPL_GASNET_SEGMENT', RUNTIME | LAUNCHER | DEFAULT),
-    ChapelEnv('  CHPL_LIBFABRIC', INTERNAL),
+    ChapelEnv('  CHPL_LIBFABRIC', INTERNAL | DEFAULT),
```

Comment:
Just based on `CHPL_GASNET_SEGMENT`, should this include  `RUNTIME` and `LAUNCHER`?

Reply:
I don't think  we need those for `CHPL_LIBFABRIC`, no.  In the case of `CHPL_COMM=gasnet` both the runtime build and the launcher build actually are dependent upon the GASNet segment, as well as the substrate.  But although for `CHPL_COMM=ofi` we need a libfabric, neither the runtime nor the launcher is sensitive to where that libfabric comes from, whether our own third-party build (which this PR is adding) or the "system" (in quotes because for the most part up until now we've actually been depending on our own private builds rather than those supplied with the OS as such).

The results of `git grep 'printchplenv.*-runtime'` seem to corroborate this, for me.

Reply:
If the user changes from `system` to `libfabric` and the versions are different, wouldn't we want to rebuild the runtime?

Reply:
Yes, my thinking is flawed.  We do what you are describing for `CHPL_HWLOC` and `CHPL_REGEXP` and possibly others, too.  I'll make that change.